README.md

```diff
@@ -5,23 +5,25 @@ We recommend starting with [Getting started](vipps-getting-started.md).
 
 ## Table of contents
 
-- [Vipps demo store](#vipps-demo-store)
-- [API documentation](#api-documentation)
-- [Technical newsletter](#technical-newsletter)
-- [Partners](#partners)
-- [Plugins](#plugins)
-- [Status pages](#status-pages)
-- [The Vipps test environment (MT)](#the-vipps-test-environment-mt)
-- [Postman](#postman)
-- [API Dashboard](#api-dashboard)
-- [Vipps API servers](#vipps-api-servers)
-- [Vipps request servers](#vipps-request-servers)
-- [Vipps API Lifecycle](#vipps-api-lifecycle)
-- [Settlements](#settlements)
-- [Vipps design guidelines](#vipps-design-guidelines)
-- [Questions?](#questions)
-
-Document version: 2.6.4.
+- [Vipps Developers](#vipps-developers)
+  - [Table of contents](#table-of-contents)
```

Comment:
The page title and the ToC itself can be skipped from the ToC.

Reply:
I will update this for all these md files here. I have a plugin that keeps the links up-to-date, so need to add  "<!-- omit in toc --> " at the end of the heading line.